NitroxClient/GameLogic/Spawning/CellRootSpawner.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using UnityEngine;
+using NitroxInt3 = NitroxModel.DataStructures.Int3;
+
+namespace NitroxClient.GameLogic.Spawning
+{
+    public class CellRootSpawner : IEntitySpawner
+    {
+        public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent, EntityCell cellRoot)
```

Comment:
Is this ever called with a parent?

Reply:
CellRoots never have a parent, if they did UWE made a big mistake somewhere